app/src/main/java/com/android/streetworkapp/ui/event/AddEvent.kt

```diff
@@ -130,6 +130,12 @@ fun AddEventScreen(
               eventViewModel.addEvent(event)
               parkViewModel.addEventToPark(event.parkId, event.eid)
 
+              // Used for the gamification feature
+              userViewModel.increaseUserScore(event.owner, 510)
```

Comment:
Improvement: instead of having hardcoded values for the score increase we could have an enum SCORE_INCREASE(val increase:Int), this would make things much more maintanable

Reply:
Done, thank you!

Reply:
It is fix, thank you!